docs/sources/write/style-guide/word-list/index.md

```diff
@@ -25,6 +25,19 @@ This page is a work in progress.
 
 ## A
 
+<!-- vale Grafana.Agentless = NO -->
+
+### agentless
+
+Grafana Alloy is a vendor-neutral distribution of the OpenTelemetry (OTel) Collector.
+It's not an agent.
+
+If you're talking about why and how to send signals directly from app to prefer no-collector to agentless.
```

Comment:
This seems unclear to me. What does it mean?

Reply:
Historically the docs described ways to send data direct without using a collector - this was called an `agentless` configuration. This `agentless` term is obsolete (and not really a great descriptive term anyway) with the introduction of Alloy.

OTel describes this a `no-collector` configuration. We prefer using `no-collector` to match upstream Otel docs.